project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md

```diff
@@ -0,0 +1,121 @@
+
+## Implementation Repositories
+
+- [anoncreds-rs] - Rust implementation of AnonCreds
+- [anoncreds-clsignatures-rs] - Rust implementation of CL Signatures used in [anoncreds-rs]. Moved from Ursa and enhanced.
+- [anoncreds-v2-rs] - Rust draft implementation of AnonCreds v2 -- **new contribution this quarter**
+
+[anoncreds-rs]: https://github.com/hyperledger/anoncreds-rs
+[anoncreds-clsignatures-rs]: https://github.com/hyperledger/anoncreds-clsignatures-rs
+[anoncreds-v2-rs]: https://github.com/hyperledger/anoncreds-v2-rs
+
+### Specifications
+
+- [AnonCreds Specification] published from the repository [anoncreds-spec]
+- [AnonCreds Methods Registry] published from the repository [anoncreds-methods-registry]
+- [AnonCreds v2.0 Specification] published from the repository [anoncreds-spec-v2]
+
+[anoncreds-spec]: https://github.com/hyperledger/anoncreds-spec
+[AnonCreds Specification]: https://hyperledger.github.io/anoncreds-spec/
+[anoncreds-methods-registry]: https://github.com/hyperledger/anoncreds-methods-registry
+[AnonCreds Methods Registry]: https://hyperledger.github.io/anoncreds-methods-registry
+[anoncreds-spec-v2]: https://github.com/hyperledger/anoncreds-spec-v2
+[AnonCreds v2.0 Specification]: https://hyperledger.github.io/anoncreds-spec-v2/
+
+### Web and Wiki Pages
+
+- [Hyperledger AnonCreds Website Home Page]
+- [Hyperledger AnonCreds Wiki Home Page]
+
+[Hyperledger AnonCreds Website Home Page]: https://www.hyperledger.org/use/anoncreds
+[Hyperledger AnonCreds Wiki Home Page]: https://wiki.hyperledger.org/display/ANONCREDS/
+
+## Project Health
+
+Contributions remain slow on the AnonCreds project itself this quarter, but
+there has been a significant uptake in conversations around the use of ZKP-based
+verifiable credentials in the AnonCreds and other VC communities. The EU has a
+requirement to use "unlinkable" (non-correlatable) verifiable credentials with
+the [EUDI Wallets]. To now, the plan to meet that requirement has not been to
+use ZKPs, but instead to issue a many instances of each credential and have the
+holder use each credential just once, or at least, each credential with just one
+verifier. Each instance is slightly different, and as a result, each verifier
+gets an unlinkable credential -- although the verifiers can correlate the
+credentials through the issuer. The reasons for the "no to ZKP" has been an
+additional requirement that each credential be bound to a hardware key held in
+the holders mobile device -- which is not possible with ZKP schemes such as CL
+Signatures used in AnonCreds v1 or BBS Signatures planned for use in AnonCreds
+v2. Some well-known cryptographers have [weighed in on that approach] and
+[presented a paper] arguing that the "batch issue" approach is ineffective. As
+well, they have proposed new scheme is being proposed that implements ZKPs using
+EDDSA hardware-based keys.  The folks involved in AnonCreds are part of that
```

Comment:
@swcurran 
Is this about the Dfinity (Google) proposal? did they mainly talk about ECDSA (and not EDDSA)?

Reply:
Good catch @JoLuPuma — thanks.  Yes — this is the Google et. al. proposal.